### ISSUE TITLE
clean up skin folder structure

### ISSUE THREAD

**Issue by yafp** [Croydon/vertical-tabs-reloaded#33]
Right now the structure of the skin folder is somehow unclear, at least to me.

**win7** and **dark** consist of several css files while **light**, **linux** and **osx** only has 1 css file which contains everything.

I guess it could make sense to give this a clean & more comparable structure. If you are fine with the basic idea i could give it a try and work on that on a small fork and offer a pull request later on.

In the same move i would put tab-border to all themes, not just the **light** one.

the name **win7** might be aswell better replaced with **win
**
Just let me know what you think about the idea.

**Comment by Croydon** [comment]
I highly appreciate your willingness to contribute, but in this case I was bit faster :smile:.

I have splitted the css of the Windows and Dark themes in serveral files because I have experimented with an idea, which I have continued to work on just yesterday. Have a look in the branch fixstuff, and, for instance, this commit: ce74eb6342868c2ba30b11ac7e372d2d6085e3ed

I also have renamed win7 to windows.
With tab border you are talking about the divider lines between tabs (#24), right? Well, I did that already as well. :)
